website/docs/components/separator/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,44 @@
+## Usage
+
+### When to use
+
+- Use to visually separate different parts or sections on a page.
+- Use it to create a clear break between paragraphs, blocks, or sections.
+
+### When not to use
+
+- Don’t use to separate headings and body text. Instead, use the proper [Display token](https://github.com/hashicorp/design-system/issues/262) to emphasize hierarchy. 
+
+## Placement
+
+We recommend using meaningful spacing between content elements, allowing users to scan and understand information quickly. Use separators sparingly and purposefully to avoid cluttering the UI and distracting users from the main content.
+
+!!! Do
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-do.png)
+
+!!!
+
+!!! Dont
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-dont.png)
+
+!!!
```

Comment:
```suggestion
!!! Dont

Don’t use to separate headings and body text. Instead, use the proper [Display token](https://github.com/hashicorp/design-system/issues/262) to emphasize hierarchy. 

![Example of a leading icon in the tag](/assets/components/separator/separator-dont.png)

!!!
```

Reply:
a couple of points: 
- what does `Display token` mean?
- are you sure the link is correct? it points to a Github issue about  Flight icons

Reply:
@didoo Fair questions, I updated the suggestion. wdyt?

Reply:
👍